libr/main/r2pm.c

```diff
@@ -600,17 +600,24 @@ static int r2pm_clone(const char *pkg) {
 	if (r_file_is_directory (srcdir)) {
 		git_pull (srcdir, 0);
 	} else {
-		char *url = r2pm_get (pkg, "\nR2PM_GIT ", TT_TEXTLINE);
-		if (url) {
-			char *dir = strchr (url, ' ');
-			if (dir) {
-				*dir++ = 0;
-				if (strcmp (dir, pkg)) {
-					R_LOG_WARN ("pkgname != clonedir");
+		char *urls = r2pm_get (pkg, "\nR2PM_GIT ", TT_TEXTLINE);
+		if (urls) {
+			int url_ct = r_str_split (urls, ',');
```

Comment:
what about using ':' instead ? like in env vars? the proposal lgtm

Reply:
iwas also thinking that || can be another good option because its not really a list, but a fallback queue

Reply:
I think , is a great character, because it cannot be used in urls